src/platform/site-wide/announcements/actions/index.js

```diff
@@ -1,10 +1,12 @@
+import localStorage from '../../../utilities/storage/localStorage';
+
 export const INIT_DISMISSED_ANNOUNCEMENTS = 'INIT_DISMISSED_ANNOUNCEMENTS';
 export const DISMISS_ANNOUNCEMENT = 'DISMISS_ANNOUNCEMENT';
 
 export const ANNOUNCEMENTS_LOCAL_STORAGE = 'DISMISSED_ANNOUNCEMENTS';
 
 function localAnnouncements(dismissedAnnouncementName) {
-  const fromLocalStorage = window.localStorage[ANNOUNCEMENTS_LOCAL_STORAGE];
+  const fromLocalStorage = localStorage.getItem(ANNOUNCEMENTS_LOCAL_STORAGE);
   let parsed = [];
```

Comment:
I always preferred to work with localStorage as a plain object instead of using the `getItem`/`setItem` functions, but as a module we need to use the latter because the data structure may not be the default `localStorage`.